pkg/action/action.go

```diff
@@ -0,0 +1,58 @@
+package action
+
+import (
+	v1api "k8s.io/client-go/pkg/api/v1"
+
+	"github.com/coreos/container-linux-update-operator/pkg/constants"
+)
+
+const (
+	// ActionsSucceeded is returned when all actions have set their configured
+	// annotation to true
+	ActionsSucceeded ActionsResult = iota
```

Comment:
I'm going to add an ActionsUnknown value. That way the zero value is not something which is used to represent an actual state, and instead represents the absence of that state. Right now it does the same thing, since everywhere that uses this value just checks `result == ActionsSucceeded`